ci-operator/config/openshift/release/openshift-release-master__nightly-4.11.yaml

```diff
@@ -81,21 +81,27 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: e2e-aws-single-node-workers
-  cron: 12 9 8,24 * *
+- as: e2e-aws-ovn-single-node-workers-serial
+  interval: 168h
   steps:
     cluster_profile: aws-2
     env:
       BASE_DOMAIN: aws-2.ci.openshift.org
+      TEST_SUITE: openshift/conformance/serial
+    observers:
+      enable:
+      - observers-resource-watch
```

Comment:
why is that necessary?

Reply:
This adds the cluster resources observer that lets us take a look at performance impacts. We do the same thing for the regular single node lanes so I figured it'd be a good idea to monitor it here as well